pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -0,0 +1,347 @@
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Click command definition for the stastic command group which includes
+cmds for managing the gathering and display of statistics that pywbemcli
+and the WBEM server can gather on the WBEM operations
+
+NOTE: Commands are ordered in help display by their order in this file.
+"""
+
+from __future__ import absolute_import, print_function
+
+import click
+
+from pywbem import Error
+
+from .pywbemcli import cli
+from ._common import warning_msg, CMD_OPTS_TXT, GENERAL_OPTS_TXT, \
+    SUBCMD_HELP_TXT
+from ._common_options import add_options, help_option
+from ._click_extensions import PywbemcliGroup, PywbemcliCommand
+
+# NOTE: A number of the options use double-dash as the short form.  In those
+# cases, a third definition of the options without the double-dash defines
+# the corresponding option name, ex. 'include_qualifiers'. It should be
+# defined with underscore and not dash
+
+# Issue 224 - Exception in prompt-toolkit with python 2.7. Caused because
+# with prompt-toolkit 2 + the completer requires unicode and click_repl not
+# passing help as unicode in options as unicode
+# NOTE: Insure that all option help attributes are unicode to get around this
+#       issue
+
+
+@cli.group('statistics', cls=PywbemcliGroup, options_metavar=GENERAL_OPTS_TXT,
+           subcommand_metavar=SUBCMD_HELP_TXT)
+@add_options(help_option)
+def statistics_group():
+    """
+    Command group for WBEM server statistics.
+
+    This command group defines commands to control the display of statistical
+    data by pywbem in conjunction with the ``--timestats`` general option which
+    enables the gathering and display of statistics.
+    Since statistical information is gathered both in the pywbem client and
+    the server, there are controls for both elements.
+
+    This command group allows enabling the gathering of statistics in a
+    WBEM server by setting the CIM_ObjectManager "GatherStasticalData" property.
+
+    In addition to the command-specific options shown in this help text, the
+    general options (see 'pywbemcli --help') can also be specified before the
+    'statistics' keyword.
+    """
+    pass  # pylint: disable=unnecessary-pass
+
+
+@statistics_group.command('reset', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_reset(context):
+    """
+    Reset the values in the client local statistics.
+
+    This command does not reset statistics in the WBEM server but
+    it does reset the statistics kept in pywbemcli on the
+    ``WBEMServerResponseTime``.
```

Comment:
Let's use the term "client statistics". That is more than just the returned server response time. Also, let's first say what it does do and then what it does not do. How about:

"This command resets the counters in the client statistics maintained by pywbemcli, including the server response times returned by the WBEM server. It does not reset the server statistics."

Reply:
Mine is not exactly the same but changed.